src/main/java/scoretag/APIDownloader.java

```diff
@@ -18,7 +18,7 @@ public static void checkAndRun(Main plugin) {
 
             try {
                 FileOutputStream fos = new FileOutputStream(placeholderApi);
-                fos.getChannel().transferFrom(Channels.newChannel(new URL("https://dl.dropboxusercontent.com/s/6rmogms1458p369/KotlinLib.jar").openStream()), 0, Long.MAX_VALUE);
+                fos.getChannel().transferFrom(Channels.newChannel(new URL("https://cloudburstmc.org/resources/kotlinlib.48/version/2442/download").openStream()), 0, Long.MAX_VALUE);
```

Comment:
The Cloudburst website doesn't allow this

Reply:
It works

Reply:
You can see for yourself

Reply:
"Server returned HTTP response code: 403 for URL: https://cloudburstmc.org/resources/kotlinlib.48/version/2442/download"